src/open_inwoner/scss/components/Typography/Link.scss

```diff
@@ -75,6 +75,8 @@
   }
 }
 
-.link + .link {
+.link
+  + .link
+  :not(.card__body--flex > a > span, .leaflet-content-details > a > span) {
```

Comment:
it seems like the linter broke off the ":not" selector so it doesn't work anymore; also this ":not" selector is a bit dangerous to use if it's just 1 or 2 exceptions; so better to correct the original style, which was faulty and not specific enough, because it dealt with the wrong exception (not your blame :-D ).
The style for the numbers in the main Navigation here should onl be:
`.primary-navigation {
  &__list-item .link.link--secondary {
    margin-left: var(--spacing-small);
  }
}`
declared outside of the ".link" curly braces.
This way all the other links everywhere will not be affected.